pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/sort/Alias.java

```diff
@@ -92,7 +91,8 @@ public static Alias of(SortBy sortBy) {
 
     public static Alias of(ParticipantColumn column) {
         Alias esAlias;
-        if (Objects.nonNull(column.getObject()) && Alias.ofOrNull(column.getObject()) != null) {
+        if (Objects.nonNull(column.getObject()) && Alias.ofOrNull(column.getObject()) != null
+                && !Alias.CL.name().equalsIgnoreCase(column.getTableAlias())) {
```

Comment:
Clinical Order is an oddly shaped column, it has an "object" field but it is not a DSS field. I tried changing the column but it broke the search, so I thought this is the fix with the smallest impact

Reply:
This method checks the display columns to determine the correct location of the data required for downloads. Each participant column contains fields that specify where the data is stored:

tableAlias: Indicates which DSM database table holds the data.
object: Points to the specific object in the JSON document where this data is located (e.g., profile, files, or dsm).
For DSS-related fields, such as date of birth, emails, and survey answers, the table alias is typically data, and the object specifies which part of the JSON document contains this data.

An exception is the Clinical Order columns. Although these also have an object field, their table alias `cl` is the identifier that should be used. The reasons behind this design choice are not fully explained, but any changes to this structure will break the filtering processes and can also have unforeseen impacts.

Reply:
Thx for adding the comment and breaking out the method. Very helpful.
This ^^ explanation even better!
You should feel free to add longer explanations like this to the code. Someone will likely benefit from it.